kusd/downloader/downloader.go

```diff
@@ -30,12 +29,12 @@ var (
 	MaxReceiptFetch = 256 // Amount of transaction receipts to allow fetching per request
 	MaxStateFetch   = 384 // Amount of node state values to allow fetching per request
 
-	MaxForkAncestry  = 3 * params.EpochDuration // Maximum chain reorganisation
-	rttMinEstimate   = 2 * time.Second          // Minimum round-trip time to target for download requests
-	rttMaxEstimate   = 20 * time.Second         // Maximum rount-trip time to target for download requests
-	rttMinConfidence = 0.1                      // Worse confidence factor in our estimated RTT value
-	ttlScaling       = 3                        // Constant scaling factor for RTT -> TTL conversion
-	ttlLimit         = time.Minute              // Maximum TTL allowance to prevent reaching crazy timeouts
+	MaxForkAncestry  = 3 * uint64(30000) // Maximum chain reorganisation (3 * params.EpochDuration - 30000)
```

Comment:
Why 30, 000? And Can we put this in a `const` somewhere so it's easy to change?

Reply:
reverted the change - added an issue to cover that field - https://github.com/kowala-tech/kcoin/issues/189 - Our epoch is shorter than Ethereum's.